src/main/java/io/nats/client/Options.java

```diff
@@ -1658,6 +1665,15 @@ public Builder useDispatcherWithExecutor() {
             return this;
         }
 
+        /**
+         * Instruct requests to turn off flush on requests.
+         * @return the Builder for chaining
+         */
+        public Builder dontForceFlushOnRequest() {

```

Comment:
personally not sure about using `dont`, maybe something like `disableForceFlushOnRequest` instead?